src/main/resources/sectiontool.properties

```diff
@@ -51,6 +51,14 @@ canvas.api.search.courses.regex=^/api/v1/accounts/1/courses\\?search_term=.*&per
 #/api/v1/accounts/self/users?search_term=instx_='+ generateCurrentTimestamp();
 canvas.api.search.user.regex = ^/api/v1/accounts/self/users\\?search_term=.*$
 
+#POST
+#/api/v1/accounts/1/users?account_id=1&user[name]=Gsilver3%20Test19&pseudonym[unique_id]=gsilver3-test19+mailinator.com&pseudonym[sis_user_id]=gsilver3-test19@mailinator.com&pseudonym[send_confirmation]=false&communication_channel[type]=email&communication_channel[address]=gsilver3-test19@mailinator.com&communication_channel[skip_confirmation]=false
+canvas.api.create.user.regex=^/api/v1/accounts/1/users\\?account_id=1&user\\[name\\]=.*&pseudonym\\[unique_id\\]=.*&pseudonym\\[sis_user_id\\]=.*&pseudonym\\[send_confirmation\\]=.*&communication_channel\\[type\\]=.*&communication_channel\\[address\\]=.*&communication_channel\\[skip_confirmation\\]=.*$
```

Comment:
So, you have a working regex now?  What was the solution?  It appears that you had to escape the backslashes used to escape the square brackets.  Is that right?  Would enclosing the values of the properties have been a solution as well?